sdk/servicebus/service-bus/src/util/runtimeInfo.ts

```diff
@@ -0,0 +1,23 @@
+// Copyright (c) Microsoft Corporation.
+// Licensed under the MIT license.
+
+import * as os from "os";
+
+/**
+ * Returns information about the platform this function is being run on.
+ * @ignore
+ * @internal
+ */
+export function getRuntimeInfo(): string {
+  const runtimeInfo = {
+    key: "Node",
+    value: process.version
+  };
+
+  const osInfo = {
+    key: "OS",
+    value: `(${os.arch()}-${os.type()}-${os.release()})`
+  };
+
+  return `${[runtimeInfo, osInfo]}`;
```

Comment:
I suspect this also returns `[object Object], [object Object]` since it's essentially calling `runtimeInfo.toString()` and `osInfo.toString()` inside the template string.

Reply:
Fixed with b3cc282

Reply:
`azsdk-js-azureservicebus/7.0.0-preview.5 Node/v12.16.0 OS/(x64-Windows_NT-10.0.18363)`